vizro-core/src/vizro/models/_components/graph.py

```diff
@@ -223,6 +242,27 @@ def build(self):
         # transparent and has no axes so it doesn't draw anything on the screen which would flicker away when the
         # graph callback is executed to make the dcc.Loading icon appear.
         description = self.description.build().children if self.description else [None]
+        defaults = {
+            "id": self.id,
+            "figure": (
+                go.Figure(
+                    layout={
+                        "paper_bgcolor": "rgba(0,0,0,0)",
+                        "plot_bgcolor": "rgba(0,0,0,0)",
+                        "xaxis": {"visible": False},
+                        "yaxis": {"visible": False},
+                    }
+                )
+            ),
+            "config": {
+                "autosizable": True,
+                "frameMargins": 0,
+                "responsive": True,
+                "modeBarButtonsToRemove": ["toImage"],
+            },
+        }
+        graph_defaults = _set_defaults_nested(self.extra, defaults)
```

Comment:
Would be nice to have a comment here on why this is different from all other components where we just do a simple merge.